modules/Microsoft.Insights/webTests/deploy.bicep

```diff
@@ -0,0 +1,130 @@
+@description('Required. Name of the webtest.')
+param name string
+
+@description('Required. User defined name if this WebTest.')
+param webTestName string
+
```

Comment:
Does the resource also support locks? If so, please add the corresponding parameters, variables & deployment as per the other modules.

Reply:
Added lock code to the file